pgdatadiff/pgdatadiff.py

```diff
@@ -19,9 +19,9 @@ def make_session(connection_string):
 
 class DBDiff(object):
 
-    def __init__(self, firstdb, seconddb, chunk_size=10000, count_only=False):
+    def __init__(self, firstdb, seconddb, schema, chunk_size=10000, count_only=False, exclude_tables=""):
```

Comment:
@andrikoz Shall we follow the `exclude_tables` pattern here instead?

```python
schema="public"
```

Reply:
Yup, I agree